model/constants.py

```diff
@@ -71,8 +71,11 @@ def moma(model, reference, linear=False):
 REMOVED_REACTIONS = 'removed-reactions'
 ADDED_REACTIONS = 'added-reactions'
 MISSING_MEASURED_REACTIONS = 'missing-measured-reactions'
-
-MESSAGE_HASH_KEYS = {GENOTYPE_CHANGES, MEDIUM, MEASUREMENTS, REACTIONS_ADD, REACTIONS_KNOCKOUT, OBJECTIVE}
+BOUNDS = 'bounds'
+CHANGED_REACTIONS = 'changed-reactions'
+REACTION_ID = 'reactions-knockout'
+MESSAGE_HASH_KEYS = {GENOTYPE_CHANGES, MEDIUM, MEASUREMENTS, REACTIONS_ADD, REACTIONS_KNOCKOUT, OBJECTIVE, BOUNDS,
+                     REACTION_ID}
```

Comment:
not sure `BOUNDS` and `REACTION_ID ` belong here since they are inside "changed reactions" object